dropwizard/service/src/main/java/org/apache/polaris/service/dropwizard/logging/QuarkusLoggingConfiguration.java

```diff
@@ -0,0 +1,30 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.apache.polaris.service.dropwizard.logging;
+
+import io.smallrye.config.ConfigMapping;
+import java.util.Map;
+
+@ConfigMapping(prefix = "polaris.log")
+public interface QuarkusLoggingConfiguration {
```

Comment:
WDYT about keeping reusable Quarkus components (link this config) in a separate module from `application.properties`?

In my experience, building a custom Quarkus server from artifacts that contain `applicaiton.properties` is a nightmare in terms of controlling where specific config options are defined.

Reply:
Yes, we'll need to split the Quarkus module into a few ones for easier integration. But I think that doing it right now would make the review process even more complicated, so let's wait until after the switch.

Reply:
SGTM